docs/v0.3/sdk/go/8g34z-get-started.md

```diff
@@ -0,0 +1,452 @@
+---
+slug: /8g34z/get-started-sdk-go
+---
+
+# Get Started with the Dagger SDK for Go
+
+## Introduction
+
+[Dagger](https://dagger.io/) is a DevOps automation engine. It enables DevOps teams to orchestrate and deliver pipelines using any supported programming language. Popular use cases include multi-stage builds, CI/CD, on-demand staging environments, machine learning pipelines, end-to-end testing, artifact scanning and signing, and more.
+
+Dagger 0.3 exposes its functionality via a low-level language-agnostic API framework and provides native language SDKs to communicate with this API. The Dagger SDK for Go is one such SDK. It enables pipeline authors to use Dagger directly from Go, without needing to learn a new language.
+
+This tutorial teaches you the basics of using Dagger in Go. You will learn how to:
+
+- Install the Dagger SDK for Go
+- Run and test a simple application with Dagger
+- Build and publish a container image of the application with Dagger
+
+## Requirements
+
+This tutorial assumes that:
+
+- You have a basic understanding of the Go programming language. If not, [read the Go tutorial](https://go.dev/doc/tutorial/getting-started).
+- You have a Go development environment with Go 1.15 or later. If not, [download and install Go](https://go.dev/doc/install).
+- You have Docker installed and running on the host system. If not, [install Docker](https://docs.docker.com/engine/install/).
+- You have a Docker Hub account. If not, [register for a free Docker Hub account](https://hub.docker.com/signup).
+
+## Step 1: Create a simple Go application
```

Comment:
```suggestion
## Step 1: Create a simple Go tool
```

Important to talk about "tool" instead of "application". To avoid confusion with the actual application, which is probably being built, tested and deployed by the tool.